link-parser/command-line.c

```diff
@@ -233,84 +230,80 @@ static int x_issue_special_command(const char * line, Command_Options *copts, Di
 			printf("%s turned %s.\n", as[j].description, (ival(as[j]))? "on" : "off");
 			return 0;
 		}
-		else
-		{
-			/* Found an abbreviated command, but it wasn't a boolean */
-			/* Replace the abbreviated command by the full one */
-			/* Basically, this just fixes !v and !h for use below. */
-			strcpy(s, user_command[k].s);
-		}
-	}
 
-	if (strcmp(s, "variables") == 0)
```

Comment:
I will merge this pull-req as is, but let me first mention a coding technique that I like that I call "indentation power reduction".  In the above, there would be `if (count == 0) return something;` and all of the rest of the code that follows can be indented one space less.  I find that that fewer indentations makes code easier to read, easier to understand. I can't quite tell if that would work here, but it seems like it might.   Anyway, its not that important, here, its just a nice trick.